cmake/KokkosConfigCommon.cmake.in

```diff
@@ -91,27 +89,23 @@ function(kokkos_check)
   endforeach()
   set(KOKKOS_CHECK_SUCCESS TRUE)
   foreach(arg ${REQUESTED_ARGS})
-    # Define variables named after the required arguments that are provided by
-    # the Kokkos install.
+    set(missing)
```

Comment:
Technically
```suggestion
    set(missing "")
```
this is safer in case someone defined it as a cache variable.
Also noting that other local variables were uppercase.  Not blocking though.

Reply:
I can do the changes, so what do you think of:
- `missing` -> `MISSING_OPTIONS`
- `_found_matching_option` -> `FOUND_MATCHING_OPTION`
- `set(MISSING_OPTIONS "")`

Reply:
Sure

Reply:
done in 6b2fec215c53e057001d0e1cf0eb523715d40447